### PR TITLE
Adding Reversed Numbers

### PR DESCRIPTION
**Pull request by DeromirNeves**
- [x] Solution to the problem ADDREV (http://www.spoj.com/problems/ADDREV/)

### CODE REVIEW

File: ADDREV/solution.ex [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  defp parse(l) do
+    String.trim(l)
+    |> String.split(" ")
+  end
+
+  defp reverse(s) do
+    String.reverse(s)
+  end
+
+  defp string_to_int(s) do
```
Comment (RenanBardy) [inline ADDREV/solution.ex]: Why don't you use a function called `format_to` or `convert_to`, then with an `atom` you specify the convertion itself?

Something like this:
```elixir
defp format_to(x, :int) do
    Integer.parse(x)
    |> elem(0)
  end

  defp format_to(x, :string) do
    Integer.to_string(x)
  end
```

File: ADDREV/solution.ex [hunk 2]
```diff
@@ -0,0 +1,32 @@
+defmodule ReverseAdder do
+  defp parse(l) do
```
Comment (RenanBardy) [inline ADDREV/solution.ex]: Can we use `_` as default to private functions?
Do you agree with that?
https://github.com/elixir-lang/elixir/blob/master/lib/elixir/pages/Naming%20Conventions.md#underscore-_foo

Comment (DeromirNeves) [reply ADDREV/solution.ex]: Updated

File: ADDREV/solution.ex (hunk 1 above)
Comment (DeromirNeves) [reply ADDREV/solution.ex]: Updated

**Pull request merged**